<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parametrs</class>
 <widget class="QWidget" name="Parametrs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="growth1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>120</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="weight1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>30</number>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="age1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
  </widget>
  <widget class="QLabel" name="growth">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;Рост (см.)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="weight">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;Вес (кг.)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="age">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;Возраст&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="face">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600; color:#ffffff;&quot;&gt;Здравствуйте, это приложение &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600; color:#ffffff;&quot;&gt;для подсчёта выпитой воды &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1181</width>
     <height>851</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>fon2.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>growth1</zorder>
  <zorder>weight1</zorder>
  <zorder>age1</zorder>
  <zorder>growth</zorder>
  <zorder>weight</zorder>
  <zorder>age</zorder>
  <zorder>face</zorder>
  <zorder>start</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
